views/home.njk

```diff
@@ -7,7 +7,7 @@
       <div class="usa-grid federalist-hero">
         <div class="usa-width-two-thirds hero-heading">
           <h1>
-            Rapidly launch a compliant website for your federal agency or office
+            Rapidly launch a compliant website for your federal agency
```

Comment:
I think "agency" alone confuses users because it makes them think they have to be representing the entire agency when they call us.

Reply:
@wslack have we heard that feedback from users directly? 

I don't mind using a term other than `agency`, but I think this generally could use some simplification given that it's the call to action and the first thing people see. Additionally, as users dig deeper into the site, they learn more and more about what is offered to them. I suggest we pick just one noun here - `agency`, `office`, `team`, or something else.

Reply:
I have heard the confusion about "can I get Federalist for just my office" directly, but not about that specific verbiage.

What about "program?"

Reply:
I like program! Let's try that and test it.